ivy_tests/test_ivy/test_functional/test_core/test_elementwise.py

```diff
@@ -2094,3 +2094,30 @@ def test_isreal(
         on_device=on_device,
         x=x[0],
     )
+
+
+@handle_test(
+    fn_tree="functional.ivy.rad2deg",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("numeric")
+    ),
+)
+def test_rad2deg(
```

Comment:
this test already exist in line 2012, I think this function has already been implemented in ivy master too, may I know where did you select this function to work on from? like the todo list, do you also have a related GitHub issue?

Reply:
yes from todo list